cmd/sonobuoy/app/pluginCache.go

```diff
@@ -274,12 +274,11 @@ func uninstallPlugin(installedDir, filename string) error {
 }
 
 func expandPath(path string) (string, error) {
-	home, err := os.UserHomeDir()
-	if err != nil {
-		return "", err
-	}
-
 	if len(path) > 0 && path[0] == '~' {
+		home, err := os.UserHomeDir()
+		if err != nil {
+			return "", err
+		}
```

Comment:
The way it was before would cause errors if HOME wasn't set even if the user was using a path that didnt include `~`